### ISSUE TITLE
Support "background image" banners

### ISSUE THREAD

**Issue by andrewatfornax** [revive-adserver/revive-adserver#370]
Add support for background image banners, or takeover banners - or some such name that is industry standard for this type of banner.

**Comment by edubuc** [comment]
It doesn't seems to be in 3.2.0 or 3.2.1 ?

**Comment by mbeccati** [comment]
@edubuc in fact the issue is still open w/ no milestone set

**Comment by edubuc** [comment]
Do you have a quick tip on how to do that in the current state of revive
adserver ?

2015-06-29 14:11 GMT+02:00 Matteo Beccati <notifications@github.com>:

> @edubuc <https://github.com/edubuc> in fact the issue is still open w/ no
> milestone set
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/revive-adserver/revive-adserver/issues/370#issuecomment-116630250>
> .
>



-- 
*Emmanuel Dubuc*
Directeur associé

edubuc@alakees.com
6 rue Sadi Carnot
93170 Bagnolet
Tél:+33 1 43 63 95 84
Mob:+33 6 22 69 36 91
Skype: emmanuel.dubuc

http://www.alakees.com

**Comment by SolutionDEVs** [comment]
Any news?

**Comment by andrewatfornax** [comment]
No, sorry, still haven't had a chance to really dig into this one.
